net-libs/nativebiginteger/nativebiginteger-0.9.36-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 Valérian Rousset
```

Comment:
Are you really sure you're the only copyright holder for this package? I see that you've taken it over after another maintainer.

Reply:
I've rewriten it to ensure that I'm, so as to have a known starting point (0.8.7 ebuild was added to git by the root commit, which can be hard to track from the old CVS). I can put the "Gentoo Authors" instead if wanted.

Reply:
Yes, 'Gentoo Authors' would be preferable. There's already a request for Council to require it for gentoo.git but I don't know whether it will pass or not.

Reply:
Done, thanks for the insight.